metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -151,15 +151,20 @@ public TableDto create(final QualifiedName name, final TableDto tableDto) {
     }
 
     private void setOwnerIfNull(final TableDto tableDto, final String user) {
-        if (!Strings.isNullOrEmpty(user)) {
-            StorageDto serde = tableDto.getSerde();
-            if (serde == null) {
-                serde = new StorageDto();
-                tableDto.setSerde(serde);
-            }
-            if (Strings.isNullOrEmpty(serde.getOwner())) {
-                serde.setOwner(user);
-            }
+        String owner = user;
```

Comment:
Shall we keep this line  `if (!Strings.isNullOrEmpty(user)) {` as the first check?